packages/markdown/components/CodeTabs/style.scss

```diff
@@ -22,45 +21,42 @@
     &::-webkit-scrollbar {
       display: none;
     }
-    > button {
+    button {
       white-space: nowrap;
       transition: .123s ease;
+      -webkit-appearance: none;
+      appearance: none;
+      display: inline-block;
+      line-height: 2;
+      padding: .5em 1em;
+      border: none;
+      background: transparent;
+      outline: none;
+      color: inherit;
+      font: inherit;
+      font-size: .75em;
+      cursor: pointer;
     }
   }
 
-  pre {
-    border-radius: 0 0 $radius $radius !important;
-    background: $bgc-pre;
-    margin-bottom: 0;
-    &:not(.CodeTabs_active) { display: none }
-  }
-
-  button {
```

Comment:
Rejiggered and further specified these unclassed element selectors (here, as well as the preceding ruleset on lines 31-36.)